pages_bots/ncc_page.py

```
@@ -0,0 +1,114 @@
+# revisions= api_new.get_revisions(title)
+
+"""
+import sys
+import os
+import configparser
+from pathlib import Path
+from ..super.S_API import bot_api
+from ..super.S_Page import super_page
+from ..super.S_Category import catdepth_new
+
+print_test = {1: True if "test" in sys.argv else False}
```

Ruff (0.11.9)

Remove unnecessary `True if ... else False`

Remove unnecessary `True if ... else False`

(SIM210)